File: src/java/WebNinja/src/test/java/webninja/selenuim/LogonPage.java
package webninja.selenuim;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class LogonPage {
    private final WebDriver webDriver;

    public LogonPage(WebDriver webDriver) {
        this.webDriver = webDriver;
    }
    
    public void setName(String name){
        webDriver.findElement(
                By.name("userLogin[username]"))
                .sendKeys(name);
    }

    public void setPassword(String password){
        webDriver.findElement(
                By.name("userLogin[password]"))
                .sendKeys(password);
    }

}

File: src/dotnet/WebNinja/WebNinja/selenium/LogonPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Remote;

namespace WebNinja.selenium
{
    public class LogonPage : FormObject
    {
        public LogonPage(IWebDriver driver) : base(driver)
        {
        }

        public string Name
        {
            set
            {
                Driver.FindElement(
                    By.Name("userLogin[username]"))
                    .SendKeys(value);
            }
        }

        public string Password
        {
            set
            {
                Driver.FindElement(
                    By.Name("userLogin[password]"))
                    .SendKeys(value);
            }
        }
    }
}